frontend/online-shop/src/services/users.service.ts

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+import { LOGIN_ENDPOIT } from "../helpers/strings";
+
+export const fetchLoginCredentials = async ({email, password}: any) => {
+    const res = await axios.post(LOGIN_ENDPOIT, {username: email, password:password});
```

Comment:
You can directly return the response here, no need to store it in a separate attribute